mcs/class/test-helpers/PlatformDetection.cs

```diff
@@ -8,7 +8,11 @@ static partial class PlatformDetection
 		public static readonly bool IsWindowsNanoServer = false;
 
 		public static bool IsWindows7 => false;
+#if MONO_COREFX
```

Comment:
Why do you need this?

Reply:
@marek-safar some tests use it in order to test "old" behavior instead of .NET Core - https://github.com/dotnet/corefx/blob/6ab836db8df5ec9af77063e45f42adccf86ec0b5/src/System.Text.RegularExpressions/tests/Regex.Ctor.Tests.cs#L310

Reply:
right but we always define it hence `public static bool IsFullFramework => false;` should be enough